### PR TITLE
move start qr code session to background queue

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**

### CODE REVIEW

**Review by oa-s — APPROVED** [review]

**Comment by hboon** [comment]
@oa-s warm fuzzy feeling writing Obj-C?:)

File: QRCodeReaderViewController/QRCodeReader.m [hunk 1]
```diff
@@ -153,14 +153,18 @@ - (void)toggleTorch
 - (void)startScanning
 {
   if (![self.session isRunning]) {
-    [self.session startRunning];
+      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
```
Comment (hboon) [inline QRCodeReaderViewController/QRCodeReader.m:156]: @oa-s I didn't check further, but this is safe to run in the background?

Comment (oa-s) [reply QRCodeReaderViewController/QRCodeReader.m:156]: should be good, xcode told to do it in the background
https://github.com/AlphaWallet/alpha-wallet-ios/issues/5749

Comment (oa-s) [reply QRCodeReaderViewController/QRCodeReader.m:156]: tested with a real devide, was good.

**Comment by oa-s** [comment]
> warm fuzzy feeling writing Obj-C?:)

damn, so hard. haven't written for long time, even syntax is hard

**Comment by hboon** [comment]
> > warm fuzzy feeling writing Obj-C?:)
> 
> damn, so hard. haven't written for long time, even syntax is hard

The syntax is less familiar, but so much simpler. Swift seems more about language features, less about app features…

**Pull request merged**